homework/03LessonThree.js

```diff
@@ -89,22 +89,22 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = [1, 3, 'chubby', null, 'niner'];
 
 // DO NOT MODIFY
-var nameArray = ['bob', 'fred', 'susan'];
+var nameArray = ['bob', 'george', 'susan'];
```

Comment:
Do not modify the variables directly.

You need to use dot or bracket notation to access and assign values to problems 10 & 11.